### PR TITLE
Update basic_views.py

### PR DESCRIPTION
**Pull request by AdamSwid**
Added /niggroMode command

### CODE REVIEW

File: src/views/basic_views.py [hunk 1]
```diff
@@ -21,6 +25,7 @@ def __init__(self, interface_layer):
             ('^/poll\s(?P<question>.+)\?\s(?P<options>.*)$', self.start_poll),
             ('^/vote\s(?P<number>\d)$', self.vote),
             ('^/endpoll$', self.end_poll),
+            ('^/niggroMode', self.niggro_mode),
```
Comment (GalPressman) [inline src/views/basic_views.py]: put $ for end line

Comment (AdamSwid) [inline src/views/basic_views.py]: FORGOT :(

Comment (GalPressman) [inline src/views/basic_views.py]: you can change it

Comment (AdamSwid) [inline src/views/basic_views.py]: you need to edit and create new pull to change?

Comment (GalPressman) [inline src/views/basic_views.py]: you change the code and push v2 for same request

File: src/views/basic_views.py [hunk 2]
```diff
@@ -58,7 +63,12 @@ def overwatch_countdown(self, message, match):
     def overwatch_hype(self, message, match):
         self.send_image('http://assets.vg247.com/current/2015/09/overwatch.jpg', message.getFrom())
         self.send_text('OVERWATCH HYPEEE!', message.getFrom())
-
+    
+    def niggro_mode
```
Comment (GalPressman) [inline src/views/basic_views.py]: (self, message, match):

File: src/views/basic_views.py [hunk 3]
```diff
@@ -58,7 +63,12 @@ def overwatch_countdown(self, message, match):
...
+        MESSAGE = NIGGRO_MODE[randint(0, len(NIGGRO_MODE) -1)]
+        self.send_image('https://pbs.twimg.com/media/Bm-Aaf-CEAAHA-P.jpg:large', message.getFrom())
```
Comment (GalPressman) [inline src/views/basic_views.py]: what is this :large?

File: src/views/basic_views.py [hunk 4]
```diff
@@ -58,7 +63,12 @@ def overwatch_countdown(self, message, match):
...
+        self.send_text(MESSAGE, message.getFrom))
```
Comment (GalPressman) [inline src/views/basic_views.py]: getFrom()

Comment (AdamSwid) [inline src/views/basic_views.py]: OK fixed everything, Commit #4

**Comment by AdamSwid** [comment]
removed :

**Comment by GalPressman** [comment]
Merged

**Pull request closed without merging**